aws/datadog_integration_role.yaml

```diff
@@ -56,72 +56,6 @@ Conditions:
       - Ref: ResourceCollectionPermissions
```

Comment:
Let's use this opportunity to make these permissions specific to Cloud Security Posture Management, and change the name from `ResourceCollectionPermissions ` to `CloudSecurityPostureManagementPermissions`

Also change the corresponding param description to: `Set this value to "true" to add permissions for Datadog's Cloud Security Posture Management product to monitor your AWS cloud resource configurations. You will also need "BasePermissions" set to "Full".`